plugins/modules/win_package.py

```diff
@@ -39,6 +39,24 @@
       or uninstalling a package.
     - This is only used for the C(msi), C(msp), and C(registry) providers.
     type: path
+  checksum:
+    description:
+      - If a I(checksum) is passed to this parameter, the digest of the
+        package will be calculated before executing it to verify that the
+        transfer completed successfully.
+    type: str
```

Comment:
Need to specify when the option was added

```suggestion
    type: str
    version_added: 2.8.0
```